decompiler/__init__.py

```diff
@@ -31,7 +31,9 @@
 from . import atldecompiler
 from . import astdump
 
-__all__ = ["astdump", "magic", "sl2decompiler", "testcasedecompiler", "translate", "util", "Options", "pprint", "Decompiler", "renpycompat"]
+__all__ = [
+    "astdump", "magic", "sl2decompiler", "testcasedecompiler", "translate", "util", "Options",
+    "pprint", "Decompiler", "renpycompat"]
```

Comment:
I think in this case (nothing after the opening bracket, an immediate linebreak, I'd prefer the closing `]` on its own line as well.